doc/docs/C++_Tutorial.md

```diff
@@ -198,26 +198,26 @@ In order to compile your code and link against the Meep library, you must link i
 
 (1) After compiling the Meep package following the instructions elsewhere, place foo.cpp in the `tests/` subdirectory, cd to the same directory, and invoke:
 
-```
+```html
```

Comment:
`html` is wrong here ... this is shell code, so it should probably be `sh`

Reply:
Unfortunately `sh` introduces unnecessary highlighting for certain code blocks. `text` produces non-highlighted text but contains no text boxes with sliders; this sometimes gives rise to sentences which awkwardly extend beyond the margins. 

Among the available [language options](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml), `html` produced no or minimal highlighting for text (i.e., non Scheme/Python/Shell). Even though this label seems inappropriate, it produces the right visual layout.

Reply:
Why not just use no label at all, then?  Seems better to use no highlighting at all than highlighting for the wrong language.

Reply:
No label also produces unnecessary highlighting. In any event, I have remove the `html` labels.